src/middlewared/middlewared/plugins/iscsi.py

```diff
@@ -400,25 +400,26 @@ class Config:
         datastore_prefix = 'iscsi_target_extent_'
         datastore_extend = 'iscsi.extent.extend'
 
-    @accepts(Dict(
-        'iscsi_extent_create',
-        Str('name', required=True),
-        Str('type', enum=['DISK', 'FILE'], default='DISK'),
-        Str('disk', default=None, null=True),
-        Str('serial', default=None, null=True),
-        Str('path', default=None, null=True),
-        Int('filesize', default=0),
-        Int('blocksize', enum=[512, 1024, 2048, 4096], default=512),
-        Bool('pblocksize'),
-        Int('avail_threshold', validators=[Range(min=1, max=99)], null=True),
-        Str('comment'),
-        Bool('insecure_tpc', default=True),
-        Bool('xen'),
-        Str('rpm', enum=['UNKNOWN', 'SSD', '5400', '7200', '10000', '15000'],
-            default='SSD'),
-        Bool('ro'),
-        register=True
-    ))
+    @accepts(
```

Comment:
Can you please revert the indentation to what it was before using rebase and force-push? This clouds the history for no gain.

We should avoid changing indentation just for the sake of change, everyone have their own preference. Or we come up with specific guidelines so everyone can follow ; )

Reply:
No worries ; ) Will have it updated right away